backend/migrations/versions/d1d6da1b2d67_add_columns_worksheet_query_result_model.py

```diff
@@ -0,0 +1,37 @@
+"""add_columns_worksheet_query_result_model
+
+Revision ID: d1d6da1b2d67
+Revises: d274e756f0ae
+Create Date: 2024-09-10 14:34:31.492186
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'd1d6da1b2d67'
+down_revision = 'd274e756f0ae'
+branch_labels = None
+depends_on = None
+
+
```

Comment:
We should also drop the now unused columns: sqlBody, queryType?

Reply:
we can drop queryType column from the table.
sqlBody and status can be kept as an reserved columns which will be used in future